pt/tutorials-and-examples/blog/part-two.rst

```diff
@@ -496,7 +496,7 @@ A seguir, vamos criar uma maneira para os usuários excluírem posts. Comece com
 uma action ``delete()`` no PostsController::
 
     function delete($id) {
-        if (!$this->request->is('post')) {
+        if (!$this->request->is('get')) {
```

Comment:
We want to only allow delete on `post`. This will throw an exception if delete is done with anything other than `GET`. I don't think we should be recommending people use `GET` for destructive actions.